givitsite/gatherer/tasks.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import unicode_literals
+from background_task import background
+from friendreq.models import ItemRequest
+from friendreq.models import ItemsFound
+from .agora import givit_main
+
+
+@background(schedule=60)
+def gatherer_task():
+    iseek_dict = {'20016': 'bed', '20009': 'closet', '20008': 'chair',
+                  '20017': 'sofa', '10006': 'fridge', '10029': 'Washing machine'}
+    item_list = givit_main()
+    counter = 0
+    for item in range(len(item_list)):
+        if len(item_list[item]) > 6:
+            print(item_list[item][6])
```

Comment:
Consider using proper logging rather then prints: https://docs.djangoproject.com/en/3.1/topics/logging/

Reply:
sound good, i will do it, issue #62